src/settings/Settings.js

```diff
@@ -210,6 +210,12 @@ export const SETTINGS = {
         default: true,
         invertedSettingName: 'hideDisplaynameChanges',
     },
+    "showCallRelatedMessages": {
+        supportedLevels: LEVELS_ROOM_SETTINGS,
+        displayName: _td('Show messages about call information'),
+        default: true,
+        invertedSettingName: 'hideCallRelatedMessages',
```

Comment:
this is only needed if migrating from a legacy setting, please remove

Reply:
done, ty!

Reply:
should we add a note on this to the documentation? (docs/settings.md)

Reply:
good idea, feel free to do it as part of this PR.